operations/agent-flow-mixin/dashboards/cluster-node.libsonnet

```diff
@@ -164,11 +172,21 @@ local filename = 'agent-cluster-node.json';
         }) +
         panel.withQueries([
           panel.newQuery(
-            expr='1 - ( \r\n    rate(cluster_transport_tx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_tx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            expr = ||| 
+              1 - ( 
+                rate(cluster_transport_tx_packets_failed_total{instance="$instance"}[$__rate_interval]) / 
+                rate(cluster_transport_tx_packets_total{instance="$instance"}[$__rate_interval])
+                )
+|||,
```

Comment:
The formatting here looks a little messed up on GitHub. Ditto below on lines 184, 254, and 263.

Reply:
This was weird to me as well, but Grizzly complains with an error like the following if using a named parameter (`expr = `) and the multiline string.
```
ERRO[2768] Error: RUNTIME ERROR: dashboards/cluster-node.libsonnet:175:20 Text block not terminated with |||
```

 I can change it to use non-named parameters in the panel.newQuery, but I feel it would become a little less readable.